ams/server/src/main/java/com/netease/arctic/server/optimizing/MetricsSummary.java

```diff
@@ -47,16 +73,28 @@ public MetricsSummary(Collection<TaskRuntime> taskRuntimes) {
         .map(TaskRuntime::getMetricsSummary)
         .forEach(
             metrics -> {
+              newFileSize += metrics.getNewFileSize();
+              newFileCnt += metrics.getNewFileCnt();
+              newDataFileCnt += metrics.getNewDataFileCnt();
+              newDataSize += metrics.getNewDataSize();
+              newDataRecordCnt += metrics.getNewDataRecordCnt();
+              newDeleteSize += metrics.getNewDeleteSize();
+              newDeleteFileCnt += metrics.getNewDeleteFileCnt();
+              newDeleteRecordCnt += metrics.getNewDeleteRecordCnt();
               rewriteDataFileCnt += metrics.getRewriteDataFileCnt();
-              reRowDeletedDataFileCnt += metrics.getReRowDeletedDataFileCnt();
```

Comment:
We should not remove `reRowDeletedDataFileCnt += metrics.getReRowDeletedDataFileCnt()`, right? The old `positionalDeleteSize` and `reRowDeletedDataFileCnt` should be treated the same way.

Reply:
`reRowDeletedDataFileCnt += metrics.getReRowDeletedDataFileCnt()` has been moved below 😂, I will adjust it back to avoid this difference.

Reply:
Done.